src/globus_cli/commands/transfer.py

```diff
@@ -33,6 +33,10 @@
 
 @command(
     "transfer",
+    opts_to_combine={
+        "include": "filter_rules",
+        "exclude": "filter_rules",
+    },
```

Comment:
I like the interface you added here. It's relatively easy to read and guess what it does.

I wish we could attach this somehow to the options themselves, rather than the command, but I don't see a cheap and easy path to doing that. At best, we'd need custom option classes to store some extra values.